back/engines/commercial/multi_tenancy/spec/tasks/clean_up_image_records_task_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# rubocop:disable RSpec/DescribeClass
+describe 'rake cl2back:clean_up_image_records' do
+  before do
+    load_rake_tasks_if_not_loaded
+    ENV['EXECUTE_CLEAN_UP_UNUSED_IMAGE_RECORDS'] = 'true'
```

Comment:
See [comment](https://github.com/CitizenLabDotCo/citizenlab/pull/3928/files#r1108535729), for why we use an ENV var here.